akka-http/src/main/scala/akka/http/scaladsl/server/RouteResult.scala

```diff
@@ -43,6 +43,17 @@ object RouteResult {
   implicit def routeToFlow(route: Route)(implicit system: ClassicActorSystemProvider): Flow[HttpRequest, HttpResponse, NotUsed] =
     Route.toFlow(route)
 
+  /**
+   * Turns a `Route` into a server function.
+   *
+   * This implicit conversion is defined here because `Route` is an alias for
+   * `RequestContext => Future[RouteResult]`, and the fact that `RouteResult`
+   * is in that type means this implicit conversion come into scope whereever
+   * a `Route` is given but a `Function[HttpRequest, Future[HttpResponse]` is expected.
+   */
+  implicit def routeToFunction(route: Route)(implicit system: ClassicActorSystemProvider): Function[HttpRequest, Future[HttpResponse]] =
```

Comment:
Was that syntax intentional? Had to lookup what that `Function` even refers to...  (it's `Predef.Function` which is an alias for `Function1`.

```suggestion
  implicit def routeToFunction(route: Route)(implicit system: ClassicActorSystemProvider): HttpRequest => Future[HttpResponse] =
```